opentasks/src/main/java/org/dmfs/tasks/EditTaskFragment.java

```diff
@@ -678,7 +678,7 @@ public void saveAndExit()
 
         if (mValues.isInsert() || mValues.isUpdate())
         {
-            if (TextUtils.isEmpty(TaskFieldAdapters.TITLE.get(mValues)))
+            if (TextUtils.isEmpty(TaskFieldAdapters.TITLE.get(mValues)) || TaskFieldAdapters.TITLE.get(mValues).trim().equals(""))
```

Comment:
I think changing to `.trim().isEmpty()` would be more performant; it simply checks if the `length` is 0.

String.isEmpty was introduced in Java 6.